build-logic/src/main/kotlin/terasology-metrics.gradle.kts

```diff
@@ -16,6 +16,29 @@ plugins {
 dependencies {
     "pmd"("net.sourceforge.pmd:pmd-core:6.15.0")
     "pmd"("net.sourceforge.pmd:pmd-java:6.15.0")
+
+    testRuntimeOnly("ch.qos.logback:logback-classic:1.2.11") {
+        because("runtime: to configure logging during tests with logback.xml")
```

Comment:
I love these `because` thingies :heart: really helps with context now and in the future